src/fileformats/ocd_file_export.cpp

```diff
@@ -1984,12 +2028,12 @@ int OcdFileExport::checkCombinedSymbol(const CombinedSymbol* combined_symbol) co
 	// The implementation must mirror exportCombinedSymbol()!
 	
 	auto num_parts = 0;  // The count of non-null parts.
-	const Symbol* parts[3] = {};  // A random access list without holes
+	const Symbol* parts[4] = {};  // A random access list without holes
```

Comment:
The magic number defining the array size is a candidate for a symbolic constant (ES.45).